### ISSUE TITLE
[iOS] No push notifications if the safe was removed and readded in case of non-owner key and owner key are imported

### ISSUE THREAD

**Issue by liliya-soroka** [safe-global/safe-ios#1066]
**Bug description**
No push notifications if the safe was removed and readded in case of non-owner key and owner key are imported
Push notifications will appear only after the app reinstallation. No any kind of push notifications
**Steps To Reproduce**
1. Add safe
2. Import owner key and non-owner key
**missed point**: receive push notification for confirmation tx ( I think that device registration happens here)
3. relaunch app -> Go to the safe settings and remove the safe
4. relaunch the app ( cold start)
5. Add safe again
6. Try all types of notification: confirmation required, incoming tx, successful/fail execution
**Current result** No notifications
**Expected Result** The user should receive allowed types of notifications

**Device & App version**
 - Device/OS: Iphone 11/14.3
 - App version: 2.13.0 (366)
 
**Environment**
prod and staging, rinkeby
**Additional info**
(Optional)

**Comment by sche** [comment]
Could not reproduce it on 2.14.0 (7)

**Comment by liliya-soroka** [comment]
I still can reproduce it in 2.14.0(10). the missed step is added

**Comment by sche** [comment]
Duplicate of https://github.com/gnosis/safe-ios/issues/1109
I propose to close this issue as in 1109 the reason of failure is clear.

**Comment by liliya-soroka** [comment]
The issue is closed in favour of the https://github.com/gnosis/safe-ios/issues/1109
